### PR TITLE
Fix #1540 wax upper limit

### LINKED ISSUE #1540: [Google Maps Support] Prevent library from fetching tiles that are out of bounds
CartoDB.js is fetching tiles for invalid coords and zoom.

The example in http://localhost:9001/examples/v4/gmaps-layergroup.html has this problem: 

screen shot 2016-12-29 at 13 22 47

CartoDB.js is requesting some invalid tiles, such as http://2.ashbu.cartocdn.com/documentation/api/v1/map/90e64f1b9145961af7ba36d71b887dd2:0/0/2/2/4.png

And Windshaft is returning:

```
{
  "errors": [
    "Coordinates out of range: 2/2/4"
  ],
  "errors_with_context": [
    {
      "type": "unknown",
      "message": "Coordinates out of range: 2/2/4"
    }
  ]
}
```

### PR DESCRIPTION
**Pull request by donflopez**
[Here is the explanation](https://github.com/CartoDB/wax/pull/11)

### CODE REVIEW

File: vendor/wax.cartodb.js [hunk 1]
```diff
@@ -3359,7 +3359,7 @@ wax.g.connector.prototype.getTileUrl = function(coord, z) {
 
     x = (x < 0) ? (coord.x % mod) + mod : x;
 
-    if (y < 0) return this.options.blankImage;
+    if (y < 0 || y > (mod - 1)) return this.options.blankImage;
```
Comment (alonsogarciapablo) [inline vendor/wax.cartodb.js]: I guess we could use `y >= 0` too.
I'm not a big fan of comments... but perhaps it makes sense to add one here (along [these lines](https://github.com/CartoDB/wax/pull/11#issue-205841475)). What do you think?

**Comment by alonsogarciapablo** [comment]
Solution looks good to me 👍 .

Tests are failing and it seems to be related to this... Could you please take a look? thanks!

**Comment by donflopez** [comment]
Sure! Done :)

**Comment by donflopez** [comment]
I checked it in staging, you can try it in ded05 @alonsogarciapablo.

**Comment by alonsogarciapablo** [comment]
🚀

**Pull request merged**